### ISSUE TITLE
[Package Issue]: IObit.Uninstaller

### ISSUE THREAD

**Issue by Edoardo66** [microsoft/winget-pkgs#160767]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

When upgrading to the new version 13.6.0.2, winget complains that the installer hash doesn't match. Previously reported, I think erroneously, the presence of a trojan in the installer

### Steps to reproduce

winget upgrade IObit.Uninstaller
or
winget install IObit.Uninstaller

### Actual behavior

Installation doesn't start with an error message (The installer hash does not match)

### Expected behavior

winget should upgrade the program to the newer version

### Environment

```raw
Gestione pacchetti Windows v1.8.1791
Copyright (c) Microsoft Corporation. Tutti i diritti sono riservati.

Windows: Windows.Desktop v10.0.22631.3737
Architettura di sistema: X64
Pacchetto: Microsoft.DesktopAppInstaller v1.23.1791.0

Directory WinGet
-----------------------------------------------------------------------------------------------------------------------
Log                                         %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalS…
Impostazioni utente                         %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalS…
Directory collegamenti portabili (utente)   %LOCALAPPDATA%\Microsoft\WinGet\Links
Directory collegamenti portabili (computer) C:\Program Files\WinGet\Links
Radice pacchetto portabile (utente)         %LOCALAPPDATA%\Microsoft\WinGet\Packages
Radice pacchetto portabile                  C:\Program Files\WinGet\Packages
Radice pacchetto portabile (x86)            C:\Program Files (x86)\WinGet\Packages
Download del programma di installazione     %USERPROFILE%\Downloads

Collegamenti
------------------------------------------------------------------------------------
Informativa sulla privacy    https://aka.ms/winget-privacy
Contratto di licenza         https://aka.ms/winget-license
Comunicazioni di terze parti https://aka.ms/winget-3rdPartyNotice
Home page                    https://aka.ms/winget
Condizioni di Windows Store  https://www.microsoft.com/en-us/storedocs/terms-of-sale

Impostazione amministrazione              Stato
------------------------------------------------------
LocalManifestFiles                        Disabilitate
BypassCertificatePinningForMicrosoftStore Disabilitate
InstallerHashOverride                     Disabilitate
LocalArchiveMalwareScanOverride           Disabilitate
ProxyCommandLineOptions                   Disabilitate
DefaultProxy                              Disabilitate
```

**Comment by Dragon1573** [comment]
![image](https://github.com/microsoft/winget-pkgs/assets/49941141/c40fceee-2145-4130-a132-563f1e75061a)

> [!TIP]
>
> This package using a vanity URL as `InstallerUrl`, https://github.com/microsoft/winget-pkgs/labels/Error-Hash-Mismatch will frequently occur when a newer version is released.

**Comment by Dragon1573** [comment]
> [!NOTE]
>
> @Edoardo66, This issue is already resolved by #160167.
>
> @stephengillie, I think we can manually close this issue as "completed".
